pkg/client/results/doc.go

```diff
@@ -17,3 +17,22 @@ limitations under the License.
 // Package results provides a low level API to extract data from a Sonobuoy
 // result archive.
 package results
+
+// Item is the central format for plugin results. Various plugin
+// types can be transformed into this simple format and set at a standard
+// location in our results tarball for simplified processing by any consumer.
+type Item struct {
```

Comment:
Not sure why this ended up in `doc.go`; I'll move it to somewhere more appropriate.